compliance_checker/suite.py

```diff
@@ -100,7 +110,22 @@ def run(self, ds, *checker_names):
 
         for checker_name, checker_class in checkers:
 
-            checker = checker_class()
+            # try to get the version of the standard from the checker after the
+            # colon
+            try:
+                standard_version = checker_name.rsplit(':', 1)[1]
+            # if no version was supplied after a colon, use None as the version
+            except IndexError:
+                # is this assignment even needed?
+                standard_version = None
+                checker = checker_class()
+            else:
+                # Otherwise use the constructor with the version number
+                # specified.  If an invalid version is specified, leave that
+                # up to the class being called to determine that and throw
+                # an exception if necessary
+                checker = checker_class(version=standard_version)
```

Comment:
I'm not entirely clear what you meant here.  This tries to pull the standard version by splitting along a delimiter (:).  If it's not there, it uses the default constructor for that specified class.